classes/class-ppc-update.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Pre-Publish Checklist.
+ *
+ * PHP version 7
+ *
+ * @category PHP
+ * @package  Pre-Publish Checklist.
+ * @author   Display Name <username@brainstormforce.com>
+ * @license  http://brainstormforce.com
+ * @link     http://brainstormforce.com
+ */
+
+if ( ! class_exists( 'PPC_Update' ) ) :
+	/**
+	 * Pre-Publish Checklist doc comment.
+	 *
+	 * PHP version 7
+	 *
+	 * @since 1.1.0
+	 * @category PHP
+	 * @package  Pre-Publish Checklist.
+	 * @author   Display Name <username@brainstormforce.com>
+	 * @license  http://brainstormforce.com
+	 * @link     http://brainstormforce.com
+	 */
+	class PPC_Update {
+		/**
+		 * Member Variable
+		 *
+		 * @var instance
+		 */
+		private static $instance;
+
+		/**
+		 * Option key for stored version number.
+		 *
+		 * @var instance
+		 */
+		private $db_version_key = '_ppc_db_version';
+
+		/**
+		 *  Initiator
+		 */
+		public static function get_instance() {
+			if ( ! isset( self::$instance ) ) {
+				self::$instance = new self();
+			}
+			return self::$instance;
+		}
+
+		/**
+		 * Constructor
+		 */
+		public function __construct() {
+
+			// Plugin updates.
+			if ( is_admin() ) {
+				add_action( 'admin_init', array( $this, 'init' ), 5 );
+			} else {
+				add_action( 'wp', array( $this, 'init' ), 5 );
+			}
+		}
+
+		/**
+		 * Implement theme update logic.
+		 *
+		 * @since 1.1.0
+		 */
+		public function init() {
+			do_action( 'ppc_update_before' );
+
+			if ( ! $this->needs_db_update() ) {
+				return;
+			}
+
+			$db_version = get_option( $this->db_version_key, false );
+
+			if ( version_compare( $db_version, '1.0.2', '<' ) ) {
+				$this->update_options_table();
+			}
+
+			$this->update_db_version();
+
+			do_action( 'ppc_update_after' );
+		}
+
+		/**
+		 * Moves the option values from ppc_checklist_data to ppc_cpt_checklist_data.
+		 *
+		 * @since 1.1.0
+		 */
+		public function update_options_table() {
+			$ppc_default_checklist_data = array(
+				'ppc_key2' => 'Featured Image Assigned',
```

Comment:
Strings need to be translation ready here too.

Reply:
__('Featured Image Assigned', 'pre-publish-checklist' ),

Added for full checklist.